--- supermin-4.1.4/helper/bin2s.pl.old	2016-01-13 11:28:11.673688319 -0500
+++ supermin-4.1.4/helper/bin2s.pl	2016-01-13 11:28:22.319412685 -0500
@@ -18,6 +18,10 @@
 print $ofh <<"EOF";
 /* This file has been automatically generated from $infile by $0 */
 
+/* Mark stack as non-executable for GNU tools. */
+\t.section .note.GNU-stack,"",%progbits
+\t.previous
+
 \t.globl\t_binary_${infile}_start
 \t.globl\t_binary_${infile}_end
 \t.globl\t_binary_${infile}_size
